app/collections/create/page.tsx

```diff
@@ -0,0 +1,19 @@
+import { CollectionForm } from "@/components/collections/collection-form";
+
+const CreateCollectionPage = async () => {
```

Comment:
I don't think this needs to be async if you aren't fetching data.

```suggestion
const CreateCollectionPage = () => {
```

Reply:
fixed